app/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Please I highly recommend sticking to the recommended [FIGMA DESIGN TEMPLATE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=TOtxhVudjc3Nx9HA-0). 😃 

_**REQUIRED FIGMA DESIGN TEMPLATE**_

![Screenshot (73)](https://user-images.githubusercontent.com/78104544/235040978-0328b9b5-63d7-4912-ba64-b8bf13cd1207.png)


_**YOUR DESIGN**_

- [ ] Please I highly suggest improving the positions of decorations which are the background images to match the required figma design. 😃 
- [ ] Please I recommend adding Medium and Angellist icons `banner-social` class. 😄 

![Screenshot (233)](https://user-images.githubusercontent.com/78104544/235041171-6fbc9079-27a1-4e5f-a9d5-8a273624bc8e.png)

Reply:
- [ ] Your project design is still missing a particular social media icon as shown below, It is a project requirement that you achieve the figma template which you selected and as such i will kindly ask that you try as much as possible to add the missing `medium icon` :+1: 
![studfent-5](https://user-images.githubusercontent.com/89055502/235203388-2113d353-bfd5-4b5c-8839-a3439d02a06d.png)